### ISSUE TITLE
[BUG] Videos can't be uploaded as media without thumbnail

### ISSUE THREAD

**Issue by EDM115** [EDM115/unzip-bot#133]
```
2023-01-05T15:13:10.804833+00:00 app[worker.1]: 2023-01-05 15:13:10,804 - WARNING - unzipper - MainThread - the first argument must be callable
2023-01-05T15:13:10.933014+00:00 app[worker.1]: 2023-01-05 15:13:10,932 - WARNING - unzipper - MainThread - the first argument must be callable
2023-01-05T15:13:10.968039+00:00 app[worker.1]: 2023-01-05 15:13:10,933 - ERROR - pyrogram.dispatcher - MainThread - [Errno 2] No such file or directory: '/app/Downloaded/854158484'
2023-01-05T15:13:10.968041+00:00 app[worker.1]: Traceback (most recent call last):
2023-01-05T15:13:10.968042+00:00 app[worker.1]:   File "/app/unzipper/modules/ext_script/up_helper.py", line 85, in send_file
2023-01-05T15:13:10.968042+00:00 app[worker.1]:     vid_duration = await run_cmds_on_cr(
2023-01-05T15:13:10.968042+00:00 app[worker.1]:   File "/app/unzipper/modules/ext_script/ext_helper.py", line 23, in run_cmds_on_cr
2023-01-05T15:13:10.968043+00:00 app[worker.1]:     return await loop.run_in_executor(None, partial(func, kwargs))
2023-01-05T15:13:10.968043+00:00 app[worker.1]: TypeError: the first argument must be callable
2023-01-05T15:13:10.968043+00:00 app[worker.1]: 
2023-01-05T15:13:10.968044+00:00 app[worker.1]: During handling of the above exception, another exception occurred:
2023-01-05T15:13:10.968044+00:00 app[worker.1]: 
2023-01-05T15:13:10.968044+00:00 app[worker.1]: Traceback (most recent call last):
2023-01-05T15:13:10.968044+00:00 app[worker.1]:   File "/usr/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 240, in handler_worker
2023-01-05T15:13:10.968045+00:00 app[worker.1]:     await handler.callback(self.client, *args)
2023-01-05T15:13:10.968045+00:00 app[worker.1]:   File "/app/unzipper/modules/callbacks.py", line 580, in unzipper_cb
2023-01-05T15:13:10.968045+00:00 app[worker.1]:     await send_file(
2023-01-05T15:13:10.968045+00:00 app[worker.1]:   File "/app/unzipper/modules/ext_script/up_helper.py", line 166, in send_file
2023-01-05T15:13:10.968046+00:00 app[worker.1]:     shutil.rmtree(full_path)
2023-01-05T15:13:10.968046+00:00 app[worker.1]:   File "/usr/lib/python3.10/shutil.py", line 715, in rmtree
2023-01-05T15:13:10.968046+00:00 app[worker.1]:     onerror(os.lstat, path, sys.exc_info())
2023-01-05T15:13:10.968047+00:00 app[worker.1]:   File "/usr/lib/python3.10/shutil.py", line 713, in rmtree
2023-01-05T15:13:10.968047+00:00 app[worker.1]:     orig_st = os.lstat(path)
2023-01-05T15:13:10.968047+00:00 app[worker.1]: FileNotFoundError: [Errno 2] No such file or directory: '/app/Downloaded/854158484'
2023-01-05T15:13:11.876185+00:00 app[worker.1]: 2023-01-05 15:13:11,876 - WARNING - unzipper - MainThread - the first argument must be callable
2023-01-05T15:14:20.306446+00:00 app[worker.1]: 2023-01-05 15:14:20,306 - WARNING - unzipper - MainThread - the first argument must be callable
2023-01-05T15:14:20.452873+00:00 app[worker.1]: 2023-01-05 15:14:20,452 - WARNING - unzipper - MainThread - the first argument must be callable
2023-01-05T15:14:20.453317+00:00 app[worker.1]: 2023-01-05 15:14:20,452 - ERROR - pyrogram.dispatcher - MainThread - [Errno 2] No such file or directory: '/app/Downloaded/1919228034'
2023-01-05T15:14:20.453318+00:00 app[worker.1]: Traceback (most recent call last):
2023-01-05T15:14:20.453318+00:00 app[worker.1]:   File "/app/unzipper/modules/ext_script/up_helper.py", line 85, in send_file
2023-01-05T15:14:20.453319+00:00 app[worker.1]:     vid_duration = await run_cmds_on_cr(
2023-01-05T15:14:20.453319+00:00 app[worker.1]:   File "/app/unzipper/modules/ext_script/ext_helper.py", line 23, in run_cmds_on_cr
2023-01-05T15:14:20.453319+00:00 app[worker.1]:     return await loop.run_in_executor(None, partial(func, kwargs))
2023-01-05T15:14:20.453319+00:00 app[worker.1]: TypeError: the first argument must be callable
2023-01-05T15:14:20.453319+00:00 app[worker.1]: 
2023-01-05T15:14:20.453320+00:00 app[worker.1]: During handling of the above exception, another exception occurred:
2023-01-05T15:14:20.453320+00:00 app[worker.1]: 
2023-01-05T15:14:20.453320+00:00 app[worker.1]: Traceback (most recent call last):
2023-01-05T15:14:20.453320+00:00 app[worker.1]:   File "/usr/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 240, in handler_worker
2023-01-05T15:14:20.453321+00:00 app[worker.1]:     await handler.callback(self.client, *args)
2023-01-05T15:14:20.453321+00:00 app[worker.1]:   File "/app/unzipper/modules/callbacks.py", line 580, in unzipper_cb
2023-01-05T15:14:20.453321+00:00 app[worker.1]:     await send_file(
2023-01-05T15:14:20.453321+00:00 app[worker.1]:   File "/app/unzipper/modules/ext_script/up_helper.py", line 166, in send_file
2023-01-05T15:14:20.453322+00:00 app[worker.1]:     shutil.rmtree(full_path)
2023-01-05T15:14:20.453322+00:00 app[worker.1]:   File "/usr/lib/python3.10/shutil.py", line 715, in rmtree
2023-01-05T15:14:20.453322+00:00 app[worker.1]:     onerror(os.lstat, path, sys.exc_info())
2023-01-05T15:14:20.453323+00:00 app[worker.1]:   File "/usr/lib/python3.10/shutil.py", line 713, in rmtree
2023-01-05T15:14:20.453323+00:00 app[worker.1]:     orig_st = os.lstat(path)
2023-01-05T15:14:20.453323+00:00 app[worker.1]: FileNotFoundError: [Errno 2] No such file or directory: '/app/Downloaded/1919228034'
```

**Comment by EDM115** [comment]
```
2023-01-05T15:30:03.915308+00:00 app[worker.1]: 2023-01-05 15:30:03,915 - WARNING - unzipper - MainThread - the first argument must be callable
2023-01-05T15:30:04.039696+00:00 app[worker.1]: 2023-01-05 15:30:04,039 - WARNING - unzipper - MainThread - Failed to decode "/app/Downloaded/854158484/extracted/ multi-medias/Spitfire Audio - Albion Collection (EDM SAMPLE PACK).torrent". The value does not represent an existing local file, HTTP URL, or valid file id.
2023-01-05T15:30:04.042546+00:00 app[worker.1]: 2023-01-05 15:30:04,039 - ERROR - pyrogram.dispatcher - MainThread - [Errno 2] No such file or directory: '/app/Downloaded/854158484'
2023-01-05T15:30:04.042547+00:00 app[worker.1]: Traceback (most recent call last):
2023-01-05T15:30:04.042548+00:00 app[worker.1]:   File "/usr/lib/python3.10/site-packages/pyrogram/file_id.py", line 228, in decode
2023-01-05T15:30:04.042548+00:00 app[worker.1]:     file_type = FileType(file_type)
2023-01-05T15:30:04.042549+00:00 app[worker.1]:   File "/usr/lib/python3.10/enum.py", line 385, in __call__
2023-01-05T15:30:04.042549+00:00 app[worker.1]:     return cls.__new__(cls, value)
2023-01-05T15:30:04.042549+00:00 app[worker.1]:   File "/usr/lib/python3.10/enum.py", line 710, in __new__
2023-01-05T15:30:04.042550+00:00 app[worker.1]:     raise ve_exc
2023-01-05T15:30:04.042550+00:00 app[worker.1]: ValueError: -60183811 is not a valid FileType
2023-01-05T15:30:04.042551+00:00 app[worker.1]: 
2023-01-05T15:30:04.042551+00:00 app[worker.1]: During handling of the above exception, another exception occurred:
2023-01-05T15:30:04.042552+00:00 app[worker.1]: 
2023-01-05T15:30:04.042552+00:00 app[worker.1]: Traceback (most recent call last):
2023-01-05T15:30:04.042553+00:00 app[worker.1]:   File "/usr/lib/python3.10/site-packages/pyrogram/utils.py", line 49, in get_input_media_from_file_id
2023-01-05T15:30:04.042553+00:00 app[worker.1]:     decoded = FileId.decode(file_id)
2023-01-05T15:30:04.042553+00:00 app[worker.1]:   File "/usr/lib/python3.10/site-packages/pyrogram/file_id.py", line 230, in decode
2023-01-05T15:30:04.042554+00:00 app[worker.1]:     raise ValueError(f"Unknown file_type {file_type} of file_id {file_id}")
2023-01-05T15:30:04.042554+00:00 app[worker.1]: ValueError: Unknown file_type -60183811 of file_id /app/Downloaded/854158484/extracted/ multi-medias/Spitfire Audio - Albion Collection (EDM SAMPLE PACK).torrent
2023-01-05T15:30:04.042555+00:00 app[worker.1]: 
2023-01-05T15:30:04.042555+00:00 app[worker.1]: During handling of the above exception, another exception occurred:
2023-01-05T15:30:04.042555+00:00 app[worker.1]: 
2023-01-05T15:30:04.042555+00:00 app[worker.1]: Traceback (most recent call last):
2023-01-05T15:30:04.042556+00:00 app[worker.1]:   File "/app/unzipper/modules/ext_script/up_helper.py", line 146, in send_file
2023-01-05T15:30:04.042556+00:00 app[worker.1]:     await unzip_bot.send_document(
2023-01-05T15:30:04.042556+00:00 app[worker.1]:   File "/usr/lib/python3.10/site-packages/pyrogram/methods/messages/send_document.py", line 178, in send_document
2023-01-05T15:30:04.042557+00:00 app[worker.1]:     media = utils.get_input_media_from_file_id(document, FileType.DOCUMENT)
2023-01-05T15:30:04.042557+00:00 app[worker.1]:   File "/usr/lib/python3.10/site-packages/pyrogram/utils.py", line 51, in get_input_media_from_file_id
2023-01-05T15:30:04.042557+00:00 app[worker.1]:     raise ValueError(
2023-01-05T15:30:04.042559+00:00 app[worker.1]: ValueError: Failed to decode "/app/Downloaded/854158484/extracted/ multi-medias/Spitfire Audio - Albion Collection (EDM SAMPLE PACK).torrent". The value does not represent an existing local file, HTTP URL, or valid file id.
2023-01-05T15:30:04.042559+00:00 app[worker.1]: 
2023-01-05T15:30:04.042559+00:00 app[worker.1]: During handling of the above exception, another exception occurred:
2023-01-05T15:30:04.042559+00:00 app[worker.1]: 
2023-01-05T15:30:04.042560+00:00 app[worker.1]: Traceback (most recent call last):
2023-01-05T15:30:04.042560+00:00 app[worker.1]:   File "/usr/lib/python3.10/site-packages/pyrogram/dispatcher.py", line 240, in handler_worker
2023-01-05T15:30:04.042560+00:00 app[worker.1]:     await handler.callback(self.client, *args)
2023-01-05T15:30:04.042561+00:00 app[worker.1]:   File "/app/unzipper/modules/callbacks.py", line 580, in unzipper_cb
2023-01-05T15:30:04.042561+00:00 app[worker.1]:     await send_file(
2023-01-05T15:30:04.042561+00:00 app[worker.1]:   File "/app/unzipper/modules/ext_script/up_helper.py", line 166, in send_file
2023-01-05T15:30:04.042562+00:00 app[worker.1]:     shutil.rmtree(full_path)
2023-01-05T15:30:04.042562+00:00 app[worker.1]:   File "/usr/lib/python3.10/shutil.py", line 715, in rmtree
2023-01-05T15:30:04.042562+00:00 app[worker.1]:     onerror(os.lstat, path, sys.exc_info())
2023-01-05T15:30:04.042562+00:00 app[worker.1]:   File "/usr/lib/python3.10/shutil.py", line 713, in rmtree
2023-01-05T15:30:04.042563+00:00 app[worker.1]:     orig_st = os.lstat(path)
2023-01-05T15:30:04.042563+00:00 app[worker.1]: FileNotFoundError: [Errno 2] No such file or directory: '/app/Downloaded/854158484'
```

**Comment by EDM115** [comment]
https://github.com/pyrogram/pyrogram/issues/1216

**Comment by EDM115** [comment]
Fixed in ffcca91ba247d9c3c7ea85990e3bffd95c92d172

**Comment by EDM115** [comment]
fixed in dbf2740aa6bcbc0feccf2ff142c8f04d2115dd3e
